Woof/Woof/Model/Owner/Owner+StorageInformation.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+extension Owner {
+    /// Provides the necessary values for work of `Current Owner`.
+    enum StorageInformation {
+        /// The name of storage associated with the owner.
+        static let storageName = "OwnerStorage"
+        /// The key associated with the owner.
+        static let ownerKey = "owner"
```

Comment:
Should that be the extension to the `KeyValueStorage`, not the `Owner`? Why the owner knows the key we should use in the storage?

smth like:

```swift
extension KeyValueStorage {
    enum StorageName {
        static let ownersDataBase = ...
    }
}
```

and

```swift
extension KeyValueStorage {
    enum Key {
        static let owner = ...
    }
}
```